internal/pkg/errors/catcher.go

```diff
@@ -200,13 +200,13 @@ func CatchResourceNotFoundError(err error, resourceId string) error {
 	return err
 }
 
-func CatchEnvironmentNotFoundError(err error, r *http.Response) error {
+func CatchOrgV2ResourceNotFoundError(err error, resourceType string, r *http.Response) error {
 	if err == nil {
 		return nil
 	}
 
 	if r != nil && r.StatusCode == http.StatusForbidden {
-		return NewWrapErrorWithSuggestions(CatchCCloudV2Error(err, r), "environment not found or access forbidden", EnvNotFoundSuggestions)
+		return NewWrapErrorWithSuggestions(CatchCCloudV2Error(err, r), fmt.Sprintf("%s not found or access forbidden", resourceType), fmt.Sprintf(OrgResourceNotFoundSuggestions, resourceType))
```

Comment:
I would expect a 404 if the resource was not found... this runs when we get a 403. Can you test that we receive a 404 if the resource doesn't exist? If the backend is returning a 403 you should let them know that their status codes are wrong.

Reply:
Tested in prod w/ `environment describe env-qwerty` and `organization describe`; both return 403 Forbidden (the latter gives an error detail explaining that the API is not available, which makes sense since it hasn't hit prod yet).

I'll make tickets (this also happens when describing non-existent Kafka clusters)